CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## V4.0.2
+
+- Add `info` prop to InfoListItem to support a third line of text.
```

Comment:
You'll need to merge this into the other changes for the changelog...just merge with dev branch:
```suggestion
- Adds `info` prop to `<InfoListItem>` to support a third line of text.
```